docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -281,35 +327,38 @@ class MyContract extends SmartContract {
 }
 ```
 
-Let's unpack what happens here. The first line of our method creates a new, empty account update for the sender account:
+To unpack what happens here, the first line of the method creates a new, empty account update for the sender account:
 
 ```ts
 let senderUpdate = AccountUpdate.create(this.sender);
 ```
 
-`AccountUpdate` is the class in o1js that represents account udpates. `AccountUpdate.create()` not only instantiates this class, but also attaches the update to the current transaction, at the same level where `create` is called. If it is called inside a `@method`, the `AccountUpdate` is created as a child (public input) of the zkApp update.
+- `AccountUpdate` is the class in o1js that represents account udpates. 
+- `AccountUpdate.create()` instantiates this class and attaches the update to the current transaction at the same level where `create` is called. 
+
+If it is called inside an `@method`, the `AccountUpdate` is created as a child (public input) of the zkApp update.
```

Comment:
This belongs into the second bullet point above; "it" refers to `AccountUpdate.create()`